client/my-sites/subscribers/subscriber-list/subscriber-row.tsx

```diff
@@ -0,0 +1,42 @@
+import { Gridicon } from '@automattic/components';
+import moment from 'moment';
+import FormCheckbox from 'calypso/components/forms/form-checkbox';
+import { SubscriberProfile } from './subscriber-profile';
+import { Subscriber } from '../types';
+
+type SubscriberRowProps = Subscriber & {
+	locale: string;
+};
+
+export const SubscriberRow = ( {
+	display_name,
+	email_address,
+	subscriptions,
+	openRate,
+	date_subscribed,
+	avatar,
+	locale,
+}: SubscriberRowProps ) => {
+	return (
+		<li className="subscriber-row row" role="row">
+			<div className="subscriber-row__checkbox-column" role="cell">
+				<FormCheckbox />
+			</div>
+			<span className="subscriber-row__profile-column" role="cell">
+				<SubscriberProfile avatar={ avatar } displayName={ display_name } email={ email_address } />
+			</span>
+			<span className="subscriber-row__subscription-type-column" role="cell">
+				{ subscriptions && subscriptions.map( ( subscription ) => <div>{ subscription }</div> ) }
+			</span>
+			<span className="subscriber-row__rate-column" role="cell">
+				{ openRate }
+			</span>
+			<span className="subscriber-row__since-column" role="cell">
+				{ moment( date_subscribed ).locale( locale ).format( 'LL' ) }
+			</span>
+			<span className="subscriber-row__menu-column" role="cell">
+				<Gridicon icon="ellipsis" size={ 24 } />
+			</span>
+		</li>
```

Comment:
Similarly, you could consider the row component as part of the subscriber-list tree, so it could look like this:
```
		<li className="subscriber-list__row" role="row">
			<div className="subscriber-list__column checkbox-column hidden" role="cell">
				<FormCheckbox />
			</div>
			<span className="subscriber-list__column profile-column" role="cell">
				<SubscriberProfile avatar={ avatar } displayName={ display_name } email={ email_address } />
			</span>
			<span className="subscriber-list__column hidden" role="cell">
				{ subscriptions && subscriptions.map( ( subscription ) => <div>{ subscription }</div> ) }
			</span>
			...
```